zrcs/zrc-7.md

```diff
@@ -0,0 +1,300 @@
+
+This cases is a mix of case 1 and case 2. Both base_uri and token_uris field are used. base_uri shouldn't an empty string and token_uris shouldn't be an empty map.
+
+Ecosystem partners should be aware of this case and implement the following rule.
+
+* Fetch a specific token URI first. If a specific token URI exists, use it as the token URI.
+* If there is no specific token URI for a token, fetch the base URI to use ```<base_uri><token_id>```.
+
+#### Example 3
+
+The base URI.
+
+```https://creatures-api.zilliqa.com/api/creature/```
+
+The ```token_uri``` state field.
+
+* 1 : "https://ipfs.zilliqa.com/ipfs/1me7ss3ARVgxv6rXqVPiikMJ8u2NLgmgszg13pY0000ZILX"
+* 2 : "https://ipfs.zilliqa.com/ipfs/1me7ss3ARVgxv6rXqVPiikMJ8u2NLgmgszg13pY0000ZILX"
+* 3 : "https://ipfs.zilliqa.com/ipfs/1me7ss3ARVgxv6rXqVPiikMJ8u2NLgmgszg13pY0000ZILX"
+
+Expected result.
+
+* 1 Metadata : ```https://ipfs.zilliqa.com/ipfs/1me7ss3ARVgxv6rXqVPiikMJ8u2NLgmgszg13pY0000ZILX```
+* 2 Metadata : ```https://ipfs.zilliqa.com/ipfs/1me7ss3ARVgxv6rXqVPiikMJ8u2NLgmgszg13pY0000ZILX```
+* 3 Metadata : ```https://ipfs.zilliqa.com/ipfs/1me7ss3ARVgxv6rXqVPiikMJ8u2NLgmgszg13pY0000ZILX```
+
+### C. Metadata Schema Description
+
+| Field name                 | Description                                                                                      |
+|----------------------------|--------------------------------------------------------------------------------------------------|
+| ```name```                 | The name of the asset. This is a required field.                                                 |
+| ```description```          | A textual description of the asset resource. This is a optional field.                           |
+| ```resource```             | A URI which points to the asset resource. This is a required field.                              |
+| ```resource_mimetype```    | A mimetype which represents the resource URI conforming to RFC 2045. This is a required field.   |
+| ```external_url```         | A URI which points to an external resource for presenting the token. This is an optional field.  |
+| ```external_description``` | A textual description which describes the entire collection. This is an optional field.          |
+| ```trait_type```           | The type name of the attribute. Can be string or integer.                                        |
+| ```value```                | The value of the attribute.                                                                      |
+| ```localization_uri```     | A URI which points to an external resource containing muti-language metadata for a specific locale This is a optional field.    |
+| ```default```              | The language of the default metadata defined in the parent object. This is an optional field.    |
+| ```locales```              | An array of supported muti-language metadata files that can be found at ```localization_uri```. This is an optional field.        |
```

Comment:
https://github.com/Zilliqa/ZRC/pull/118#issuecomment-1004606491
Please note the correct structure and fix this.